### PR TITLE
Fix for connecting to new mysql version, added new stuff

### PR DESCRIPTION
**Pull request by DrEthan77**
Updated drivers that allows streama to connect to mysql, added expiring user toggle in edit users tab so you dont have to delete them you can just expire them, and added a new role called 'trusted users' which allows you to control who can download the movie files if that is enabled.

### CODE REVIEW

**Comment by DrEthan77** [comment]
for some reason whenever you remove the role that i added from someone it removes all of the roles

File: grails-app/conf/application.yml [hunk 1]
```diff
@@ -16,8 +16,8 @@ environments:
     development:
         dataSource:
             dbCreate: update
-            driverClassName:  'com.mysql.jdbc.Driver'
-            url: jdbc:mysql://localhost/streama
+            driverClassName:  'com.mysql.cj.jdbc.Driver'
```
Comment (HannahRy) [inline grails-app/conf/application.yml]: @DrEthan77 Why have you changed the driver class? Is that a typo?

Comment (DrEthan77) [reply grails-app/conf/application.yml]: screen shot 2018-09-29 at 5 51 07 pm
nope, the old driver is depreciated

Comment (HannahRy) [reply grails-app/conf/application.yml]: Okay cool.

**Comment by DrEthan77** [comment]
nvm that is a bug with the roles in general

**Comment by HannahRy** [comment]
This looks fine to merge, we should look at that role bug though.

&nbsp;

In the future please make separate commits for the changes. IE: One commit for:

- The DB driver and related change in the app yml
- The user download changes
- Expire users

They can all be in one PR.

**Pull request merged**